blog/_posts/2016-09-28-almost-done-jammin.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Almost done jammin'
+layout: post
+created_at: Mon Sep 25 2016
```

Comment:
can you make the created_at date the publishing date (same date as in the filename and permalink).